bundles/org.openhab.automation.jrubyscripting/src/main/resources/OH-INF/config/config.xml

```diff
@@ -108,6 +113,34 @@
 			<advanced>true</advanced>
 		</parameter>
 
+		<parameter name="bundle_gemfile_path" type="text" required="false" groupName="bundler">
+			<label>Gemfile Path</label>
+			<description><![CDATA[Path to Gemfile relative to <tt>OPENHAB_CONF/automation/ruby/</tt>.
+				An absolute path can be used.
+			  If this file exists, <tt>require 'bundler/setup'</tt> will be inserted for all scripts.
+				Defaults to "<tt>Gemfile</tt>" when not specified.
+				]]></description>
+			<default>Gemfile</default>
+			<advanced>true</advanced>
+		</parameter>
+
+		<parameter name="bundle_gemfile_content" type="text" multiple="true" required="false" groupName="bundler">
+			<label>Gemfile Content</label>
+			<description><![CDATA[Literal content of bundler's Gemfile.
+				This will be used when the Gemfile doesn't exist in the Gemfile path above.
+				]]></description>
+			<advanced>true</advanced>
+		</parameter>
+
+		<parameter name="bundle_install" type="boolean" required="true" groupName="bundler">
```

Comment:
this configuration parameter is unnecessary with the auto_install Bundler setting above

Reply:
With the removal of this, and using `auto_install: true` how could we provide an option to _not_ auto install (and just do `Bundler.setup` only) to speed up startup? It would skip the need to check against source (rubygems.org).

I haven't tested what would happen with a system without an internet connection when bundler install runs.

Reply:
I've decided to keep it and run bundler install / update on startup. That would speed up the execution of the first script because the installation is already done.

Reply:
But instead of `bundle_install` parameter, I kept the old `check_update` and use it for bundler too.